tests/test-conf.yaml

```diff
@@ -4,7 +4,7 @@ dispatcher:
     dispatcher_url: 0.0.0.0
     dispatcher_port: 8001
     dispatcher_service_url: http://localhost:8001
-    sentry_url: "https://2ba7e5918358439485632251fa73658c@sentry.io/1467382"
+    sentry_url: "https://2703d13672264153a19e86f2cf5a415b@sentry.io/5729456"
```

Comment:
hm, did you accidentially store your real URL?

Reply:
yes, reverting now

Reply:
yes, but making a commit replacing it does not actually remove the token from the git history.

You will be always able to access it by specific commit, since history is preserved:

https://github.com/oda-hub/dispatcher-app/blob/117c0d82184cc6ab4ac86e2a96d7c9daa0773c8f/tests/test-conf.yaml

There are special tools to purge something from the entire tree, but it will require rewriting history, and a force-push, which is unfortunate.
Also, the token has already been exposed in the public, and there are robots specifically searching for these on github.
I think the safest and easiest way is to expire this particular token in the sentry interface, and create a new one.

Reply:
I will just re-create the project...

Reply:
It's a known issue, many people end up doing it at some point. 
Recreating project does it, but sentry specifically supports expiring tokens for this or similar reasons.